adder/single/dag_service.go

```diff
@@ -18,7 +18,7 @@ import (
 
 var errNotFound = errors.New("dagservice: block not found")
 
-var logger = logging.Logger("localdags")
+var logger = logging.Logger("singledags")
```

Comment:
Need to update `logging.go` too.

Reply:
Made this change in the latest commit, but interestingly this log is never used.